commands/InjectedCommand.java

```diff
@@ -1,52 +1,50 @@
 package org.usfirst.frc4904.standard.commands;
 
+import edu.wpi.first.wpilibj2.command.CommandBase;
 
-import edu.wpi.first.wpilibj.command.Command;
-
-public abstract class InjectedCommand extends Command {
-	private final Command previous;
+public abstract class InjectedCommand extends CommandBase {
+	private final CustomCommand previous;
 	
-	public InjectedCommand(Command previous) {
+	public InjectedCommand(String name, double timeout, CustomCommand previous) {
```

Comment:
take in a commandbase, not customCommand

Reply:
also, don't bother implementing a constructor with timeout

Reply:
Done, please check again :)